src/components/Reservations.jsx

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { Link } from 'react-router-dom';
+import swal from 'sweetalert';
+import { FaRegWindowClose } from 'react-icons/fa';
+import { useCurrentUserQuery } from '../api/authLog';
+import {
+  useDeleteReservationMutation,
+  useGetReservationsQuery,
+} from '../api/reservationsData';
+import './css/reservation.css';
+import Sidebar from './Sidebar';
+import MobileMenu from './MobileMenu';
+
+function Reservations() {
```

Comment:
- [ ] When I add a reservation like this (without the city), it works and the db is populated with a reservation with no city:

![image](https://user-images.githubusercontent.com/57421823/235855154-a19539aa-e058-44d4-b3bc-3b13643e5c20.png)

As a result, in "My reservations" the booking appears with no city:

![image](https://user-images.githubusercontent.com/57421823/235855241-d8b2422d-0e95-4890-8c1d-0c6353d855e8.png)

Also, the database table field is empty:

![image](https://user-images.githubusercontent.com/57421823/235855613-504247d2-ab71-472b-98e3-c0ca79a4ff64.png)


Kindly, add some validation so your users are not permitted to leave the "City" field form blank.

That way, the UX will be improved, and db fields won't be empty. 💪